akka-remote/src/test/scala/akka/remote/artery/RemoteConnectionSpec.scala

```diff
@@ -51,23 +51,24 @@ class RemoteConnectionSpec extends ArteryMultiNodeSpec("akka.remote.retry-gate-c
       val localSystem = newRemoteSystem()
 
       val localPort = port(localSystem)
+      val localHost = address(localSystem).host.get
       muteSystem(localSystem)
 
       val localProbe = new TestProbe(localSystem)
       localSystem.actorOf(TestActors.echoActorProps, "echo")
 
-      val remotePort = temporaryServerAddress(udp = true).getPort
+      val remotePort = temporaryServerAddress(localHost, udp = true).getPort
```

Comment:
We should adjust these tests so that they select upd/tcp port based on the Artery transport.
We already run them like that in https://jenkins.akka.io:8498/job/akka-artery-tcp-remote-tests/
```
sbt -Dakka.remote.artery.transport=tcp akka-remote/test akka-remote-tests/test
```

Reply:
Aha will also do that